nativelib/src/main/scala/scala/scalanative/runtime/ieee754tostring/ryu/RyuDouble.scala

```diff
@@ -694,7 +696,9 @@ object RyuDouble {
 
   // format: on
 
-  @noinline def doubleToString(
+  // @deprecated("Use doubleToChars() instead, as it requires no additional String/Array allocations", "0.5.0")
+  @noinline
```

Comment:
See comment about no-longer-relevant `@inline` above 'toChars` below.

Sorry, I entered the comment out-of-order.

Reply:
Done in https://github.com/scala-native/scala-native/pull/2917/commits/3c79f9080e1018e4ae5ddea35494567767049ed4